bundles/framework/mydata/instance.js

```diff
@@ -0,0 +1,312 @@
+import './Tile';
+import './Flyout';
+import './service/ViewService';
+
+/**
+ * @class Oskari.mapframework.bundle.mydata.MyDataBundleInstance
+ *
+ * Main component and starting point for the personal data functionality.
+ *
+ * See Oskari.mapframework.bundle.mydata.MyDataBundle for bundle definition.
+ */
+Oskari.clazz.define('Oskari.mapframework.bundle.mydata.MyDataBundleInstance',
+
+    /**
+     * @method create called automatically on construction
+     * @static
+     */
+
+    function () {
+        this.core = null;
+        this.sandbox = null;
+        this.started = false;
+        this.template = null;
```

Comment:
Remove unused variables:
```suggestion
        this.sandbox = null;
        this.started = false;
```